### PR TITLE
moving to aliased import

### PR DESCRIPTION
**Pull request by alexreardon**

### CODE REVIEW

File: src/view/drag-drop-context/app.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // @flow
 import React, { useEffect, useRef, type Node } from 'react';
+import invariant from 'tiny-invariant';
 import { bindActionCreators } from 'redux';
 import { Provider } from 'react-redux';
-import { useMemoOne, useCallbackOne } from 'use-memo-one';
+import { useMemo, useCallback } from 'use-memo-one';
```
Comment (TrySound) [inline src/view/drag-drop-context/app.jsx]: Why? They may be confused with original react hooks.

Comment (alexreardon) [reply src/view/drag-drop-context/app.jsx]: It works nicer with `eslint-plugin-react-hooks` which matches on `useMemo` and `useCallback`

Comment (alexreardon) [reply src/view/drag-drop-context/app.jsx]: It is up to you though. You can still use `import { useMemoOne } from 'use-memo-one'`

Comment (TrySound) [reply src/view/drag-drop-context/app.jsx]: Ah, okay. Didn't think about lint rule.

**Pull request merged**